Bottom: 5055ae349471137cea427674220eac8e8569c085
Top:    64f94e70aed8bc56563e2d32cfdd3f9f5f4f0c13
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2008-11-10 18:58:07 -0500

Refresh of jds-lirc-mce2

---

diff --git a/drivers/input/ir/ir-mceusb2.c b/drivers/input/ir/ir-mceusb2.c
index 898267a..fad46cd 100644
--- a/drivers/input/ir/ir-mceusb2.c
+++ b/drivers/input/ir/ir-mceusb2.c
@@ -685,7 +685,7 @@ static void usb_remote_disconnect(struct usb_interface *intf)
 
 	ir->usbdev = NULL;
 	input_ir_destroy(ir->input);
-	input_free_device(ir->input);
+	input_unregister_device(ir->input);
 
 	usb_kill_urb(ir->urb_in);
 	usb_free_urb(ir->urb_in);
